R/robustbayesianmetaanalysis.R

```diff
@@ -360,49 +357,49 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
-  jasp_table$addColumnInfo(name = "mean",   title = gettext("Mean"),   type = "number")
-  jasp_table$addColumnInfo(name = "median", title = gettext("Median"), type = "number")
-  jasp_table$addColumnInfo(name = "lowerCI",title = gettext("Lower"),  type = "number", overtitle = CI_overtitle)
-  jasp_table$addColumnInfo(name = "upperCI",title = gettext("Upper"),  type = "number", overtitle = CI_overtitle)
+  jaspTable$addColumnInfo(name = "terms",  title = "",                type = "string")
+  jaspTable$addColumnInfo(name = "mean",   title = gettext("Mean"),   type = "number")
+  jaspTable$addColumnInfo(name = "median", title = gettext("Median"), type = "number")
+  jaspTable$addColumnInfo(name = "lowerCI",title = gettext("Lower"),  type = "number", overtitle = overtitleCI)
+  jaspTable$addColumnInfo(name = "upperCI",title = gettext("Upper"),  type = "number", overtitle = overtitleCI)
   
   if (individual) {
-    jasp_table$addColumnInfo(name = "error", title = gettext("MCMC error"), type = "number")
-    jasp_table$addColumnInfo(name = "ess",   title = gettext("ESS"),        type = "integer")
-    jasp_table$addColumnInfo(name = "rhat",  title = gettext("Rhat"),       type = "number")
+    jaspTable$addColumnInfo(name = "error", title = gettext("MCMC error"), type = "number")
+    jaspTable$addColumnInfo(name = "ess",   title = gettext("ESS"),        type = "integer")
+    jaspTable$addColumnInfo(name = "rhat",  title = gettext("Rhat"),       type = "number")
   }
   
   
-  if (is.null(results_table)) 
-    return(jasp_table)
+  if (is.null(resultsTable)) 
+    return(jaspTable)
   
   # fill rows
-  for (i in c(1:nrow(results_table))[rownames(results_table) %in% c("mu", "tau")]) {
-    temp_row <- list(
-      terms    = .robmaCoefNames(rownames(results_table)[i], add_info),
-      mean     = results_table[i, "Mean"],
-      median   = results_table[i, "Median"],
-      lowerCI  = results_table[i, if(individual) ".025" else as.character(.5 - options[["results_CI"]] / 2)],
-      upperCI  = results_table[i, if(individual) ".975" else as.character(.5 + options[["results_CI"]] / 2)]
+  for (i in c(1:nrow(resultsTable))[rownames(resultsTable) %in% c("mu", "tau")]) {
+    tempRow <- list(
+      terms    = .robmaCoefNames(rownames(resultsTable)[i], add_info),
+      mean     = resultsTable[i, "Mean"],
+      median   = resultsTable[i, "Median"],
+      lowerCI  = resultsTable[i, if(individual) ".025" else as.character(.5 - options[["resultsCI"]] / 2)],
+      upperCI  = resultsTable[i, if(individual) ".975" else as.character(.5 + options[["resultsCI"]] / 2)]
     )
     if (individual) {
-      temp_row[["error"]] <- results_table[i, "MCMC error"]
-      temp_row[["ess"]]   <- results_table[i, "ESS"]
-      temp_row[["rhat"]]  <- results_table[i, "Rhat"]
+      tempRow[["error"]] <- resultsTable[i, "MCMC error"]
+      tempRow[["ess"]]   <- resultsTable[i, "ESS"]
+      tempRow[["rhat"]]  <- resultsTable[i, "Rhat"]
```

Comment:
don't you want to call this "rHat"?

Reply:
this is taking values computed by the RoBMA package, so it needs to use naming specified there

Reply:
oh, you mean on the right side, yes, I can change that